morph/morphemes.py

```diff
@@ -125,15 +125,16 @@ def find_class(self, cmodule, cname):
         return pickle.Unpickler.find_class(self, cmodule, cname)
 
 square_brackets_regex = re.compile(r'\[[^\]]*\]')
-round_brackets_regex = re.compile(r'（[^）]+）')
+# support for either type of parenthesis.. not sure what would be faster... written like this
+# or split into two regular expressions, or have an or in the regular expression
+# (r'（[^）]+）') # regular expression for wide parenthesis
+round_brackets_regex = re.compile(r'[\(（][^）\)]+[）\)]') 
 
 def getMorphemes(morphemizer, expression, note_tags=None):
     if cfg('Option_IgnoreBracketContents'):
-        if square_brackets_regex.search(expression):
```

Comment:
I think the 'if' conditions were added intentionally to reduce new string object allocations if they weren't needed.

Reply:
nvm, I see you made that change in the following patch.